rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/widget/CovidImageViewFactoryTest.java

```diff
@@ -26,6 +30,12 @@ public void setUp() throws Exception {
         covidImageViewFactory = new CovidImageViewFactory();
     }
 
+    @Test
+    public void testGetViewsFromJson() throws Exception {
+        List<View> views = covidImageViewFactory.getViewsFromJson("step1", RDTApplication.getInstance(), null, new JSONObject(), null);
+        Assert.assertEquals(1, views.size());
```

Comment:
Could we also verify that an image is set?

Reply:
yes, i have added another null check.

Reply:
It should not be null.

Reply:
i have added mocked bitmap image

Reply:
I was expecting a check that the image is set and is not null?

Reply:
set image from `widgetArgs` and added a check to make sure image is not null